Gudschinsky taiwan ryan

Airhunter8 opened the pull request.

Airhunter8 commented:
Hmm I don't see the 0/0 on my end could you show me where.  And this is not a completable game, it is like sudan where you just keep playing it.

AaronHemphill commented:
The game is looking really good. Here are the pending items I'm seeing...

# 1
First, when complete, create a PR for your branch into summer-interns-2025, not into main.

# 2
Remove back arrow from .xml layout

# 3
Set up an audio instruction file in the games tab and then test the updated layout. (In all other games, the button will display when the audio file is present and not display if absent.)

# 4
Once I click on first tile, I can't click on the image to hear the word audio anymore. It eventually comes back (I think after finishing the word?). Fix so that image click always plays word audio regardless of position in sequence of play.

# 5
The button widths are uneven.

# 6
The games menu button and the word textbox are overlapping.

# 7
What is the 0/0 for?

# 8
For the bottom word box, either display initially with a random theme color (0 to 4) or wait for the user to click on the bottom word box and then update the box from gray to a random theme color

Airhunter8 commented:
I changed up the base, removed the arrow, changed the box sizes, I tried to change the audio functions but audio is not fully working in general on my computer rn. removed some things from xml,
